posts/2024-05-06-check-cfg.md

```diff
@@ -52,6 +52,20 @@ fn win() {}
 
 ![cargo-check](../../../../images/2024-05-06-check-cfg/cargo-check.svg)
 
+## Expecting static cfgs
+
+*UPDATE: This section was added with the release of nightly-2024-05-19.*
+
+Some crates might use cfgs, like `loom`, `fuzzing` which are always statically known at compile time. For those cases, Cargo provides via the `[lints]` table a way to statically declare those cfgs as expected.
+
+It is done through the special `check-cfg` config under `[lints.rust.unexpected_cfgs]`:
+
+*`Cargo.toml`*
+```toml
+[lints.rust]
+unexpected_cfgs = { level = "warn", check-cfg = ['cfg(loom)', 'cfg(fuzzing)'] }
+```
+
 ## Custom cfgs and build scripts
```

Comment:
Should we update this section to not mention cfg's from `RUSTFLAGS`, leaving it to only those set via `build.rs`?

Reply:
If your referring to the `tokio_unstable` cfg, I can see the argument for not wanting to mix them with `build.rs`, but in that case wouldn't it be better to move the `In Cargo point-of-view` note above in the newly created section "Expecting static cfgs"?

Reply:
I was referring to

> Some crates use custom cfgs that they **either expected from the environment (`RUSTFLAGS`or other means)** or is enabled by some logic 
in the crate `build.rs`.

But yes, the general structure is off.  "Custom cfgs and build scripts" was written to be top-level but now we have a "dynamic" and a "static" section without updating other parts, like "In Cargo's point-of-view".

Reply:
I reworked the flow of those 2 sections. Let me know what you think.